server_installation/topics/operator/command-options.adoc

```diff
@@ -0,0 +1,29 @@
+
+[[_command-options]]
+=== Command options for managing custom resources
+
+After you create a custom request, you can edit it or delete using the `oc` command.
```

Comment:
Maybe there would be good to mention that the Keycloak operator custom resource is created in namespace/project specified by user.

Reply:
@andymunro I do not know is you observed this comment. So I am just assigning it to you. Thank you for your help. :-)